sdk/src/signer.rs

```diff
@@ -85,6 +85,11 @@ pub trait Signer {
     fn direct_cose_handling(&self) -> bool {
         false
     }
+
+    /// Returns a list of dynamic assertions that should be included in the manifest.
+    fn dynamic_assertions(&self) -> Vec<Box<dyn DynamicAssertion>> {
```

Comment:
I'm wondering if it makes sense to have this fn on the non-async `Signer` trait. Do we anticipate a use case where they would be supported / needed?

Reply:
We don't plan on it, but I don't see the harm in having it here.